### PR TITLE
Support wp argument in reste command to rm WP default dir

### PR DESCRIPTION
**Pull request by lucatume**
[Screencast](https://drive.google.com/open?id=1KYXB7VN4Na6JlNczAdnIzlBHKil5eqA1&authuser=luca%40tri.be&usp=drive_fs) 

This PR extends the `tric reset` command to support the `wp` argument.

The command `tric reset` would remove `tric` run configuration, the `.env.tric.run` file; the `tric reset wp` command will, instead, remove (recursively) the default WordPress directory used by `tric` in `/_wordpress`.

The command will not do the same for a custom set wp directory.

Changelog and version follow #71

### CODE REVIEW

File: src/commands/reset.php [hunk 1]
```diff
@@ -21,4 +56,4 @@
 	exit( 1 );
 }
 
-echo light_cyan( 'Done' );
+echo light_cyan( ' done' );
```
[the remark this answers is not in the document]
Comment (lucatume) [reply src/commands/reset.php:59]: yes, to have a Docker CLI like output.

[the remark this answers is not in the document]
Comment (sc0ttkclark) [reply src/commands/reset.php:59]: Was just curious because it seemed to be the old 'Done' way on other areas.

File: src/utils.php [hunk 2]
```diff
@@ -558,3 +558,43 @@ function upper_snake_case( $string ) {
...
+	if ( is_file( $dir ) || is_link( $dir ) ) {
+		return unlink( $dir );
+	}
+
+	$files = new \RecursiveIteratorIterator (
+		new \RecursiveDirectoryIterator(
+			$dir,
+			\RecursiveDirectoryIterator::SKIP_DOTS ),
+		\RecursiveIteratorIterator::CHILD_FIRST
+	);
```
Comment (bordoni) [inline src/utils.php:583]:
```suggestion
	$files = new \RecursiveIteratorIterator (
		new \RecursiveDirectoryIterator(
			$dir,
			\RecursiveDirectoryIterator::SKIP_DOTS
		),
		\RecursiveIteratorIterator::CHILD_FIRST
	);
```

File: src/utils.php [hunk 3]
```diff
@@ -558,3 +558,43 @@ function upper_snake_case( $string ) {
...
+	}
+
+	if ( is_file( $dir ) || is_link( $dir ) ) {
+		return unlink( $dir );
+	}
+
+	$files = new \RecursiveIteratorIterator (
+		new \RecursiveDirectoryIterator(
+			$dir,
+			\RecursiveDirectoryIterator::SKIP_DOTS ),
```
Comment (borkweb) [inline src/utils.php]:
```suggestion
			\RecursiveDirectoryIterator::SKIP_DOTS
		),
```

**Review by borkweb — APPROVED** [review]
Excellent addition!

One small tweak and this is good to go.

**Pull request merged**